grove/cache/retryinggetter.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/apache/trafficcontrol/grove/web"
 
 	"github.com/apache/trafficcontrol/lib/go-log"
+	librfc "github.com/apache/trafficcontrol/lib/go-rfc"
```

Comment:
Kind of a nitpick, but I'm not a fan of aliases. IMO first-party libraries should have good names that don't need aliases. Could we just use `rfc` here?

Reply:
no, because that shadows a grove package by the same name

I'd like to probably try to shunt that entire package into the lib, but I don't wanna do that as part of this PR.

Reply:
Ah, I see, this file already has `grove/rfc`. How much work would it be to move all of `grove/rfc` to `lib/go-rfc`?

Reply:
Sorry, replied before I saw your reply.

Yeah, that's fine.